### ISSUE TITLE
objc:nsurlsession target does not seem to support boolean values in postData payload

### ISSUE THREAD

**Issue by thboileau** [Kong/httpsnippet#65]
Hello,

I set my HAR request as follow:

``` json
{
  "headers": [
    {
      "name": "Host",
      "value": "example.com"
    },
    {
      "name": "Accept",
      "value": "application/json"
    },
    {
      "name": "Content-Type",
      "value": "application/json"
    }
  ],
  "bodySize": 160,
  "postData": {
    "text": "{\n  \"active\" : false\n}",
    "mimeType": "application/json"
  },
  "httpVersion": "HTTPS/1.1",
  "method": "POST",
  "url": "https://example.com/sample/"
}
```

Then I launch the following command line:

`httpsnippet /tmp/sources/httpsnippet/test.json --target objc --client nsurlsession`

I get the following exception:

```
/usr/local/lib/node_modules/httpsnippet/src/targets/objc/helpers.js:65
        return '@"' + value.replace(/"/g, '\\"') + '"'
                            ^
TypeError: Object false has no method 'replace'
    at Object.module.exports.literalRepresentation (/usr/local/lib/node_modules/httpsnippet/src/targets/objc/helpers.js:65:29)
    at Object.module.exports.literalRepresentation (/usr/local/lib/node_modules/httpsnippet/src/targets/objc/helpers.js:61:63)
    at Object.module.exports.nsDeclaration (/usr/local/lib/node_modules/httpsnippet/src/targets/objc/helpers.js:37:24)
    at module.exports (/usr/local/lib/node_modules/httpsnippet/src/targets/objc/nsurlsession.js:59:29)
    at /usr/local/lib/node_modules/httpsnippet/src/index.js:200:19
    at Array.map (native)
    at HTTPSnippet.convert (/usr/local/lib/node_modules/httpsnippet/src/index.js:199:33)
    at iterator (/usr/local/lib/node_modules/httpsnippet/bin/httpsnippet:83:24)
    at /usr/local/lib/node_modules/httpsnippet/node_modules/async/lib/async.js:246:17
    at /usr/local/lib/node_modules/httpsnippet/node_modules/async/lib/async.js:122:13
```

Version of node.js: 0.10.32
Version of httpsnippet: 1.16.2

**Comment by thibaultcha** [comment]
Hi,

Thanks for reporting, #67 is a proposed fix.

**Comment by ahmadnassri** [comment]
released in `v1.16.3`

**Comment by thboileau** [comment]
thanks
